types/jsdom/index.d.ts

```diff
@@ -41,7 +41,7 @@ export class JSDOM {
      * Behind the scenes, a jsdom Window is indeed a VM context.
      * To get access to this ability, use the runVMScript() method.
      */
-    runVMScript(script: Script): void;
+    runVMScript(script: Script): any;
```

Comment:
`unknown` might be more correct, however that requires TS 3.0. And making this mirror the node typings makes more sense to me